firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift

```diff
@@ -98,26 +105,23 @@ class OnboardingMultipleChoiceButtonView: UIView, Themeable {
 
         NSLayoutConstraint.activate(
             [
-                imageView.topAnchor.constraint(equalTo: containerView.topAnchor),
-                imageView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor),
-                imageView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor),
                 imageView.widthAnchor.constraint(equalToConstant: UX.Measurements.imageWidth),
                 imageView.heightAnchor.constraint(equalToConstant: UX.Measurements.imageHeight),
 
-                titleLabel.topAnchor.constraint(equalTo: imageView.bottomAnchor, constant: 8),
-                titleLabel.centerXAnchor.constraint(equalTo: imageView.centerXAnchor),
-
-                checkboxView.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 4),
-                checkboxView.centerXAnchor.constraint(equalTo: imageView.centerXAnchor),
                 checkboxView.widthAnchor.constraint(equalToConstant: UX.Measurements.checkboxDimensions),
                 checkboxView.heightAnchor.constraint(equalToConstant: UX.Measurements.checkboxDimensions),
-                checkboxView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+
+                stackView.topAnchor.constraint(equalTo: containerView.topAnchor),
+                stackView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor),
+                stackView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+                stackView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor),
 
                 button.topAnchor.constraint(equalTo: containerView.topAnchor),
-                button.leadingAnchor.constraint(equalTo: containerView.leadingAnchor),
+                button.leadingAnchor.constraint(equalTo: imageView.leadingAnchor),
                 button.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
-                button.trailingAnchor.constraint(equalTo: containerView.trailingAnchor),
+                button.trailingAnchor.constraint(equalTo: imageView.trailingAnchor),
 
+                containerView.widthAnchor.constraint(equalToConstant: 80),
```

Comment:
nit: add magic constant to UX enum / struct?

Reply:
exactly. good point